llvm/lib/Target/PowerPC/PPCISelLowering.cpp

```diff
@@ -860,6 +860,7 @@ PPCTargetLowering::PPCTargetLowering(const PPCTargetMachine &TM,
       setOperationAction(ISD::FEXP2, VT, Expand);
       setOperationAction(ISD::FSIN, VT, Expand);
       setOperationAction(ISD::FCOS, VT, Expand);
+      setOperationAction(ISD::FTAN, VT, Expand);
```

Comment:
Do we need to handle STRICT_FTAN too?

Reply:
It's already been handled by common logic, and there's already been test for constrained tan in vector-constrained-fp-intrinsics.ll